### PR TITLE
Support multiple md5 hashes of the PackageApplication script to support multiple Xcode versions

### PR DESCRIPTION
**Pull request by rbroek**
Added the hash of Xcode 8's PackageApplication script to support using the legacy build api with gym and Xcode 8.

### CODE REVIEW

**Comment by tsheaff** [comment]
I can confirm that this has fixed the issue I was experiencing. I would definitely recommend a merge.

File: gym/lib/gym/xcodebuild_fixes/package_application_fix.rb [hunk 1]
```diff
@@ -11,16 +11,16 @@ def patch_package_application
...
 
           path = File.join(Helper.gem_path("gym"), "lib/assets/package_application_patches/PackageApplication_MD5")
-          expected_md5 = File.read(path)
+          expected_md5_hashes = File.read(path).split("\n")
 
           # If that location changes, search it using xcrun --sdk iphoneos -f PackageApplication
           package_application_path = "#{Xcode.xcode_path}/Platforms/iPhoneOS.platform/Developer/usr/bin/PackageApplication"
 
-          UI.crash!("Unable to patch the `PackageApplication` script bundled in Xcode. This is not supported.") unless expected_md5 == Digest::MD5.file(package_application_path).hexdigest
+          UI.crash!("Unable to patch the `PackageApplication` script bundled in Xcode. This is not supported.") unless expected_md5_hashes.include? Digest::MD5.file(package_application_path).hexdigest
```
Comment (KrauseFx) [inline gym/lib/gym/xcodebuild_fixes/package_application_fix.rb]: Could you add `() after the `.include?(...)`

**Comment by KrauseFx** [comment]
This looks great 👍 
Do you know what has changed in the script itself?

**Comment by rvandenbroek-aa** [comment]
@KrauseFx, they now print a warning at the start of the script indicating that the script is deprecated:

```
--- /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/PackageApplication	2016-02-24 03:46:41.000000000 +0100
+++ /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/PackageApplication	2016-07-14 11:48:33.000000000 +0200
@@ -12,10 +12,12 @@
 use File::Temp qw/ tempdir :POSIX /;
 use File::Basename;
 use File::Path qw/ remove_tree /;
 use Cwd qw/ chdir /;

+print "\n\n\nwarning: PackageApplication is deprecated, use `xcodebuild -exportArchive` instead.\n\n\n";
+
 $|=1;   # don't buffer stdout

 my $program = $0;

 my %opt = ();
```

File: gym/lib/gym/xcodebuild_fixes/package_application_fix.rb (hunk 1 above)
Comment (rvandenbroek-aa) [inline gym/lib/gym/xcodebuild_fixes/package_application_fix.rb]: @KrauseFx, done. Thanks for checking out the pull request!

**Comment by KrauseFx** [comment]
This is great, thanks for investigating and providing the patch 👍

**Comment by rbroek** [comment]
Thanks for accepting and thanks for all the work on fastlane!

**Comment by kjacobs-aa** [comment]
Thanks @rbroek !!!

**Pull request merged**